include/boost/graph/hawick_circuits.hpp

```diff
@@ -153,13 +153,14 @@ namespace hawick_circuits_detail
     public:
         hawick_circuits_from(Graph const& graph, Visitor& visitor,
             VertexIndexMap const& vim, Stack& stack, ClosedMatrix& closed,
-            VerticesSize n_vertices)
+            VerticesSize n_vertices, unsigned int const maxLength)
```

Comment:
Don't `const` value parameters.
```suggestion
            VerticesSize n_vertices, unsigned int maxLength)
```